### PR TITLE
Change web server

### PR DESCRIPTION
**Pull request by mareklevv**
Summary of changes:
- add '2019' to license headers
- change Waitress web server to CherryPy
- fix all style issues

### CODE REVIEW

File: ie_serving/server/start.py [hunk 1]
```diff
@@ -57,4 +57,7 @@ def serve(models, max_workers: int=1, port: int=9000):
             for model in models:
                 models[model].update()
     except KeyboardInterrupt:
+        print("dupa")
```
Comment (dtrawins) [inline ie_serving/server/start.py]: ?

File: ie_serving/server/rest_service.py [hunk 2]
```diff
@@ -125,7 +130,19 @@ def create_rest_api(models):
...
-    app.add_route('/v1/models/{model_name}/versions/{model_version}:predict', predict)
+    app.add_route('/v1/models/{model_name}/versions/{model_version}/metadata',
+                  get_model_meta)
+    app.add_route('/v1/models/{model_name}:predict', predict)
+    app.add_route('/v1/models/{model_name}/versions/{model_version}:predict',
+                  predict)
     return app
+
+
+def start_web_rest_server(models):
```
Comment (dtrawins) [inline ie_serving/server/rest_service.py]: why this function is not included in start.py instead along with create_rest_api?

File: ie_serving/server/rest_service.py [hunk 3]
```diff
@@ -125,7 +130,19 @@ def create_rest_api(models):
...
+    d = PathInfoDispatcher({'/': create_rest_api(models)})
+    server = WSGIServer(('0.0.0.0', 5555), d)
```
Comment (dtrawins) [inline ie_serving/server/rest_service.py]: why it is not using the rest-port parameter?

File: ie_serving/server/start.py [hunk 4]
```diff
@@ -39,7 +42,7 @@ def initialize_tf():
     pass
 
 
-def serve(models, max_workers: int=1, port: int=9000):
+def serve(models, rest_thread=None, max_workers: int=1, port: int=9000):
```
Comment (mzegla) [inline ie_serving/server/start.py]: I don't see usage of `rest_thread` parameter in this function. What is the purpose of it?

**Pull request merged**